frontend/src/components/input-output/configure-ds/ConfigureDs.jsx

```diff
@@ -39,6 +39,9 @@ function ConfigureDs({
   const { sessionDetails } = useSessionStore();
   const { setAlertDetails } = useAlertStore();
   const handleException = useExceptionHandler();
+  const updateSessionDetails = useSessionStore(
+    (state) => state.updateSessionDetails
+  );
```

Comment:
**Suggestion**: @mohamed-siddhiq Can this be initialized as the following?

`const { updateSessionDetails } = useSessionStore();
`
This way it will be more readable.

Reply:
@tahierhussain Addressed your comments.